src/App.js

```diff
@@ -4,7 +4,7 @@ import Hidden from '@material-ui/core/Hidden';
 import "./index.css";
 // importing pages 
 import Index from './pages/home/index'
-import Team from './pages/team'
+import Team from './pages/team/index'
```

Comment:
if there is an `index.js` file inside a folder then you can use default import from there and not add the the `index.js` file in the import path. 
```js
import Team from './pages/team'
```
would do just remove the previous files.

Reply:
Ohk let me do it in next pr